test/lib/translate.js

```diff
@@ -0,0 +1,33 @@
+import { h } from 'preact';
+import translate from '../../src/lib/translate';
+import { Text } from '../../src/components/text';
+
+describe('translate', () => {
```

Comment:
this isn't a complete testing of `translate`, mostly missing out on the `plural` test cases, but those are covered already in the testing for `<Text />`.